scripts/dockerimages/scripts/rancheros-install

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -e
+
+usage()
+{
+cat <<EOF
+Usage: 
+  rancheros-install [options]
+Options:
+  -c cloud-config file
+      needed for SSH keys.
+  -d device
+  -f [ DANGEROUS! Data loss can happen ] partition/format without prompting
+  -p partition disk.
+        This will run fdisk and create one large ext4 partition.
+  -t install-type:
+        virtualbox-iso
+        amazon-ebs
+        rancher-upgrade
+  -v os-installer version.
```

Comment:
It is required, its used to pull in the container. For a default, I'm thinking latest.